### DISCUSSION TITLE
Paid for copilot but not activating

### DISCUSSION THREAD

**Discussion by NikhilSehgal123** [community/community#30506] [Copilot Conversations]
Hi, I paid for the yearly plan for co-pilot (which is $100), but it is not activating and keeps telling me I need to upgrade as per below:
![Screenshot 2022-08-23 at 07 39 50](https://user-images.githubusercontent.com/49030309/186113652-073685f1-556f-404c-81b8-636a905d41f9.png)

I have also checked my GitHub account and it is active.

Any idea how to solve this?

**Comment by davecheney** [comment]
@NikhilSehgal123 You are running version 1.31, the latest is version 1.42. https://marketplace.visualstudio.com/items?itemName=GitHub.copilot

**Reply by NikhilSehgal123** [reply]
![Screenshot 2022-08-23 at 09 55 02](https://user-images.githubusercontent.com/49030309/186116407-cc75660a-0c0d-43b2-88aa-4de7dbb0c03a.png)

@davecheney I am on the latest version but it still is causing an issue

**Reply by NikhilSehgal123** [reply]
@davecheney sorry I had to reload the extension, it worked! Thanks for all your help

**Reply by davecheney** [reply]
Glad you got it sorted out.

**Reply by dmpeletronic** [reply]
Also, dont forget to update your VSCode.
I had the same problem here, so I updated vscode and copilot.

**Comment by marcoscannabrava** [comment]
I ran into the same issue and I tried everything :hot_face: but it finally worked out.
Reinstalled Copilot, uninstalled it, installed Nightly Copilot, reloaded VSCode, upgraded VSCode, reloaded extension and then... it worked :grin:

**Reply by davecheney** [reply]
@marcoscannabrava im sorry you are having difficulties, if the problem reoccurs could you please paste the relevant parts of View -> Output; select _GitHub Copilot_ from the pulldown.

**Reply by marcoscannabrava** [reply]
Thanks, at every step of the way the extension output was: "Please upgrade your account"

**Comment by peshak2008** [comment]
I ran into the same issue. I was using an older version of the extension, I tried to install and reinstall a few times but it was always installing an older version of the extension. Then I realized I am running an older version of vscode, so I updated it, then this allowed the last copilot version to be installed. This took care of the activation problem.

**Reply by davecheney** [reply]
@peshak2008 were you able to determine you were running an older version from the View -> Output; select GitHub Copilot 
 pulldown?

**Reply by peshak2008** [reply]
@davecheney from the extension tab in vscode, where the extension version is shown beside the name of the extension. I tried to also force install another version by select install a different version, but the highest I could see was 1.31, until I updated vscode and automatically the 1.42 copilot got installed
